integration_tests/test_staking_cache.py

```
@@ -0,0 +1,873 @@
+    for node_idx in range(5):
+        node_cli = cronos.cosmos_cli(node_idx)
+        cache_size = [-1, 0, 1, 2, 3][node_idx]
+
+        # Get unbonding delegations for all delegators
+        total_count = 0
+        for delegator_addr in unique_delegators:
+            try:
+                unbonding = node_cli.get_unbonding_delegations(delegator_addr)
+                count = len(unbonding) if unbonding else 0
+                total_count += count
+            except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)